tests/models/bloom/test_tokenization_bloom.py

```diff
@@ -36,11 +36,10 @@ class BloomTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
 
     def setUp(self):
         super().setUp()
-        tokenizer = BloomTokenizerFast.from_pretrained("bigscience/tokenizer")
+        tokenizer = BloomTokenizerFast.from_pretrained("bigscience/tokenizer", **self.special_tokens_map)
         tokenizer.save_pretrained(self.tmpdirname)
 
     def get_rust_tokenizer(self, **kwargs):
-        kwargs.update(self.special_tokens_map)
```

Comment:
Those special tokens are (now) filled in the `tokenizer_config.json` of `"bigscience/tokenizer"`